webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -58,19 +42,19 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
             experimentId: parseInt(task.id),
             state: RunState.Active,
           });
-          updateExperimentLocally(exp => ({ ...exp, state: RunState.Active }));
+          if (onComplete) onComplete();
```

Comment:
just for the record, we are switching local updating of the experiments to a simpler and safer request for new data. :+1:  

should we let the caller know what action was completed? (onComplete) we could leave for when we'd use that.

Reply:
totally agree, let's add the event types when we need them.